File: IT/Interfacce grafiche/Esercizi/Python/esercizio1.py
# file esercizio1.py

try:
	from tkinter import Frame
except ImportError:
	from Tkinter import Frame

frame = Frame(height=100, width=150)

frame.pack()
# frame.master.geometry("50x50")
frame.master.update_idletasks()

# "100x100"
fmt = "{}x{}".format(150, 100-10)
frame.master.geometry(fmt)

# print 
# print "winfo_rootx():", frame.master.winfo_rootx()
# print "winfo_rooty():", frame.master.winfo_rooty()

frame.mainloop()

File: IT/1/Variabili, operatori, strutture di controllo/Esercizi/Java/Esercizio1.java
/* file Esercizio1.java */

public class Esercizio1 {
	public static void main(String [] args) {
		int a = 5;
		int b = 10;

		System.out.println("a = " + a + ", b = " + b);

		int c = a + b;
		a = a + b;

		System.out.println("a = " + a);
		System.out.println("b = " + b);
		System.out.println("c = " + c);
	}
}

